i18n.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  locales: ['en', 'he'],
+  defaultLocale: 'he',
+  pages: {
+    '*': ['common'],
+    '/': ['home'],
+    '/pre-form': ['pre-form'],
```

Comment:
What is this /pre-form? copied from another project? remove that.

Reply:
removed